src/styles/StyledLoader.tsx

```diff
@@ -0,0 +1,27 @@
+import styled, { keyframes } from "styled-components";
```

Comment:
change request: please move this styled component to src/components folder. Rename file StyledLoader=>Loader.